### ISSUE TITLE
Returns 500 without explanation

### ISSUE THREAD

**Issue by joeyciechanowicz** [pa11y/pa11y-webservice#70]
In a couple of cases the webservices returns an `HTTP 500` error without explanation.

Notably when you request a task with `lastres: true` and there is no last result, the web services simply replies with `500`.

**Comment by ahmedansari153** [comment]
I'm experiencing this issue actively. Any updates to handle this?

**Comment by danyalaytekin** [comment]
Thanks for bumping this issue @ahmedansari153 and sorry for the `500`s.  Is your context the same as the one described by @joeyciechanowicz?:

> when you request a task with lastres: true and there is no last result

**Comment by ahmedansari153** [comment]
Yeah, I'm experiencing pretty much an identical issue. I have over 1200 pages and it seems to bring my server to a crawl when lastres is set to true. @danyalaytekin

**Comment by ahmedansari153** [comment]
I also noted that this ticket could be related @danyalaytekin https://github.com/pa11y/pa11y-dashboard/issues/142

**Comment by danyalaytekin** [comment]
Thanks @ahmedansari153 for the detail and that link, which will be useful.
